### PR TITLE
DHFPROD-9128: Success Transition and Merge Documents

### PR DESCRIPTION
**Pull request by briantang**
#### Checklist: 
```diff
- Note: do not change the below
```

- ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Ran cypress tests on firefox locally
  

- ##### Reviewer:

- [x] Reviewed Tests
- [x] Added to Release Wiki

### CODE REVIEW

File: marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/steps/MergingStepController.java [hunk 1]
```diff
@@ -96,9 +96,22 @@ public ResponseEntity<JsonNode> validateMergingStep(@PathVariable String stepNam
         return ResponseEntity.ok(MasteringService.on(getHubClient().getFinalClient()).validateMergingStep(stepName, view, entityPropertyPath));
     }
 
+    @RequestMapping(value = "/merge", method = RequestMethod.PUT)
+    @Secured("ROLE_readMerging")
+    public ResponseEntity<JsonNode> mergeDocument(@RequestParam List<String> mergeURIs, @RequestParam String flowName, @RequestParam(required = false)  String stepNumber, @RequestParam(required = false, defaultValue = "false") Boolean preview, @RequestParam(required = false, defaultValue = "{}") String options) {
```
Comment (rahulvudutala) [inline marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/steps/MergingStepController.java:101]: @briantang There is already another mergeDocument method with the same signature and implementation. You can remove this.

**Review by rahulvudutala — CHANGES_REQUESTED** [review]

File: marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/steps/MergingStepController.java (hunk 1 above)
Comment (rahulvudutala) [reply marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/steps/MergingStepController.java]: I removed the method and force pushed the changes

**Review by ryanjdew — DISMISSED** [review]

**Comment by briantang** [comment]
Looking into some minor test/BE issue with the team.

**Review by ryanjdew — APPROVED** [review]

**Review by AkshayBajajML — DISMISSED** [review]

**Review by rahulvudutala — APPROVED** [review]

**Comment by briantang** [comment]
Couple remaining failures are unrelated except for one tweak I pushed. This should pass on the next cycle.

**Review by rahulvudutala — APPROVED** [review]

**Review by AkshayBajajML — APPROVED** [review]

**Pull request merged**